logstash-output-neo4j.gemspec

```diff
@@ -0,0 +1,34 @@
+Gem::Specification.new do |s|
+
+  s.name            = 'logstash-output-neo4j'
+  s.version         = '0.9.1'
```

Comment:
Not really, I did some previous releases before merging in here, so even if not perfect, it have to stay like this.